docs/integration_tests_with_helios_solo.md

```diff
@@ -0,0 +1,53 @@
+Since Helios 0.8.662, [helios-solo][] can also be run directly from within an
```

Comment:
should bump this to whatever the next release is after the few in-progress improvements/PRs are merged